.github/workflows/lint.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         echo ${{ github.event.pull_request.base.sha }}
         echo '${{ steps.file_changes.outputs.files}}'
 
-    - name: Run qmk info
+    - name: Run qmk lint
```

Comment:
cant mark it as a suggestion, but you might also want to change the job name on line 9, to 
```yaml
jobs:
  lint:
`